packages/environment-check/src/translations/env-check.i18n.json

```diff
@@ -48,6 +48,7 @@
         "destinationsNotFound": "Couldn't find destination {{deepDiveDestination}} in list of {{destNumber}} destinations(s)",
         "endpointsNotFound": "Couldn't find {{deepDiveEndpoint}} in list of {{sysNumber}} SAP system(s)",
         "getTransportRequestsoNotAvailable": "Unable to retrieve available Transport Requests",
+        "guidedAnswersLink": "Guided Answers troubleshooting guide available at https://ga.support.sap.com/dtp/viewer/index.html#/tree/3046/actions/45995:45996:50742:46000:57266",
```

Comment:
The referencced link needs to be cleaned up too! 
It's really difficult to follow unless you know how curl and shell scripting works. It should be a simple command that can be copied and pasted.

Reply:
Thank you, duly noted ✔️